File: app/src/main/java/com/padmitriy/android/currcalc/di/SerializerModule.java
package com.padmitriy.android.currcalc.di;

import com.google.gson.Gson;
import com.google.gson.GsonBuilder;
import dagger.Module;
import dagger.Provides;

@Module
public class SerializerModule {

    @Provides
    public Gson provideApiGson() {
        return new GsonBuilder()
                .create();
    }
}

File: app/src/main/java/com/padmitriy/android/currcalc/di/SerializerModule.kt
package com.padmitriy.android.currcalc.di

import com.google.gson.Gson
import com.google.gson.GsonBuilder
import dagger.Module
import dagger.Provides

@Module
class SerializerModule {

    @Provides
    fun provideApiGson(): Gson {
        return GsonBuilder()
            .create()
    }
}
